### ISSUE TITLE
BarterDEX goes white on OSX if you leave it open

### ISSUE THREAD

**Issue by jacobdstephens** [KomodoPlatform/BarterDEX#173]
Not sure what's up or if anyone else has this issue, but the screen goes white if I leave BarterDEX running and my machine goes to sleep.

**Comment by siulynot** [comment]
did you try to change the theme?

**Comment by jacobdstephens** [comment]
Nope. Just left it running. Seems like something in isn't getting refreshed on the UI after a certian amount of up-time.

**Comment by siulynot** [comment]
Can you share a screenshot?

**Comment by jacobdstephens** [comment]
Sure. Let me know if this works. I think I had about a 12-18 hrs of uptime before this happened. <img width="1282" alt="screen shot 2018-03-26 at 11 11 37 am" src="https://user-images.githubusercontent.com/1172907/37924473-ca56a344-30e6-11e8-9707-9623ddebc685.png">

**Comment by siulynot** [comment]
Can you press ctrl - r and see if this fixes it?

**Comment by jacobdstephens** [comment]
That looks like it fixed it. Thank you!
